demos/common/pom.xml

```diff
@@ -16,6 +16,12 @@
       <version>1.3</version>
     </dependency>
 
+	<dependency>
+      <groupId>dk.alexandra.fresco</groupId>
+      <artifactId>spdz</artifactId>
+      <version>${fresco-version}</version>
+      <type>test-jar</type>
+    </dependency>
```

Comment:
Why are we depending on the test jar in demos?

Reply:
We moved some dummy/test classes from spdz/main into spdz/test (eg. dummy storage, fake tripple generation). These are however used for the demos.

Reply:
We moved a few classes into the test part of the code since they had no place within main. In particular the DummyDataSupplier. Thus, we need the test jar to use this within the demos.

Reply:
Having thought this through, eliminating dummy data supplier from the spdz is the right way to go. However then you need to do a lot more work. Including building submaven projects for each supplier.

My suggestion is to move the dummy data suppler back into spdz. It is similar to the dummyprotocol that also is tolerated in the core

Reply:
I see your point - It would be easier to keep the dummy supplier within spdz. Also makes it possible to test applications without having to import the test jar.